components/Typography/src/heading.scss

```diff
@@ -19,4 +19,18 @@
 .utrecht-heading-6 {
   /* TODO: Move to container element, make spacing optional */
   --utrecht-space-around: 1;
+
+  font-family: var(--denhaag-typography-sans-serif-alternate);
+}
```

Comment:
Wouldn't it instead be possible to specify `utrecht.heading-6.font-family` in the design tokens JSON?

Reply:
@Robbert which tokens file? The utrecht or THe Hague